api/sheet.go

```diff
@@ -51,6 +103,9 @@ type SheetRaw struct {
 	Name       string
 	Statement  string
 	Visibility SheetVisibility
+	Source     SheetSource
+	Type       SheetType
+	Payload    string
```

Comment:
nit: add a comment about payload is in the format of SheetVCSPayload.

Reply:
Added: `// Payload is in the json string format of SheetVCSPayload.`